### ISSUE TITLE
Change instructions to link to @linbusbobcat's elementary installer app

### ISSUE THREAD

**Issue by peteruithoven** [aroman/freya-on-a-mac#32]
There seems to be a *Using Create elementary Installer* app available: 
https://elementary.io/nl/docs/installation#creating-an-installation-medium
Could we add this to step 9?

**Comment by aroman** [comment]
that app is a fork of @sdaitzman's [app](https://github.com/sdaitzman/elementary-thumbdrive-creator), which we do link in step 9. I haven't looked into the difference between the two — @sdaitzman, what do you think about changing the link to the [forked version](https://github.com/sdaitzman/elementary-thumbdrive-creator)?

**Comment by sdaitzman** [comment]
Basically, the difference is @linusbobcat has been nice enough to maintain and update their fork! We should change the link :smiley:

**Comment by aroman** [comment]
:+1:

**Comment by linusbobcat** [comment]
@aroman here are some of the changes I included:

- The removal of the ISO conversion process which isn't required since elementary OS and Ubuntu use hybrid ISOs (correct me if I'm wrong). Various online docs didn't recommend doing it either, including the Arch Wiki.
- It uses pure Applescript instead of Automator.
- Localization support (with various translations made by the community).
- Bug fixes.
- Notification support on OS X Mavericks and higher.
- A build-in tool for reformating resulting drives on OS X El Capitan and above. There's currently a bug on that OS X version that prevents doing this.

I originally wanted to contribute to the project, although I wasn't sure whether the drastic UI changes would be accepted, upon moving it to Applescript, it became kind of unmergeable.

I also wrote those Mac docs, although some difficulties led to the changes being commited by lewisgoddard.
